framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -601,21 +601,21 @@ void wifi_prov_cb_in_app(es_wifi_prov_data_s *event_data)
 	if (event_data->authtype >= NONE_AUTH && event_data->authtype <= WPA2_PSK) {
 		switch (event_data->authtype) {
 		case WEP:
-			things_strncpy(p_info->auth_type, "WEP", strlen("WEP"));
+			things_strncpy(p_info->sec_type, "WEP", strlen("WEP"));
```

Comment:
You can use define without assigning a memory (#define SEC_TYPE_WEP)

Reply:
ok.